composer.json

```diff
@@ -32,7 +32,7 @@
     "require-dev": {
         "friendsofphp/php-cs-fixer": "^2.19 || ^3.8",
         "mf2/mf2": "^0.5.0",
-        "phpstan/phpstan": "^1.11",
+        "phpstan/phpstan": "~1.12.2",
```

Comment:
Even patch versions can sometimes introduce breakage but [pinning the minor version](https://getcomposer.org/doc/articles/versions.md#tilde-version-range-) will reduce the change a bit.